lib/ae_mdw/contract.ex

```diff
@@ -430,9 +443,9 @@ defmodule AeMdw.Contract do
     end
   end
 
-  defp contract_init_args(contract_pk, tx_rec) do
+  defp contract_init_args(contract_pk, tx_rec, mod \\ :aect_create_tx) do
     with {:ok, {type_info, _compiler_vsn, _source_hash}} <- get_info(contract_pk),
-         call_data <- :aect_create_tx.call_data(tx_rec),
+         call_data <- apply(mod, :call_data, [tx_rec]),
```

Comment:
Can we use `mod.call_data(tx_rec)` instead to help the compiler throw errors?

Reply:
done